kafka-connect-s3/src/main/java/io/confluent/connect/s3/S3SinkConnectorConfig.java

```diff
@@ -1164,7 +1182,7 @@ public String toString() {
     };
 
     public static String[] names() {
-      BehaviorOnNullValues[] behaviors = values();
+      IgnoreFailBehavior[] behaviors = values();
```

Comment:
nit: 
```suggestion
      IgnoreOrFailBehavior[] behaviors = values();
```

Reply:
> nit:

@snehashisp I have incorporated above feedback with [this](https://github.com/confluentinc/kafka-connect-storage-cloud/pull/558/commits/4ac8dccf815e34d4f8540499c55ab60eedeaf3b7) commit.